sunkit_pyvista/plotter.py

```diff
@@ -279,7 +287,24 @@ def plot_field_lines(self, field_lines, **kwargs):
             grid = self._coords_to_xyz(field_line.coords.ravel())
             field_line_mesh = pv.StructuredGrid(grid[:, 0], grid[:, 1], grid[:, 2])
             color = {0: 'black', -1: 'tab:blue', 1: 'tab:red'}.get(field_line.polarity)
+            field_line_mesh.add_field_array([color], 'color')
             self.plotter.add_mesh(field_line_mesh, color=color, **kwargs)
             field_line_meshes.append(field_line_mesh)
 
         self._add_mesh_to_dict(block_name='field_lines', mesh=field_line_meshes)
+
+    def save(self, filepath):
+        """
+        Adds all of the meshes in the current dictionary
+        to a :class:`~pyvista.core.MultiBlock`.
+
+        Parameters
+        ----------
+        filepath : `str`
+            Name of the file to save as, should have vtm as an extension.
+        """
+        for objects in self.all_meshes:
+            for meshes in self.all_meshes[objects]:
+                self.mesh_block.append(meshes)
```

Comment:
Why would we want to append this if we want to save it?

Reply:
because we can only save a single multiblock right?

Reply:
My question is why does it need to be a class attribute?

Reply:
Ah yeah, it doesn't have to be actually, my bad. I'll change that